providers/infrastructure-vsphere/v1.4.1/cconly/base.yaml

```diff
@@ -1421,6 +1427,21 @@ spec:
               - level: Metadata
                 omitStages:
                   - "RequestReceived"
+  - name: controlPlaneTaint
+    enabledIf: '{{ not .controlPlaneTaint }}'
+    definitions:
+    - selector:
+        apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+        kind: KubeadmControlPlaneTemplate
+        matchResources:
+          controlPlane: true
+      jsonPatches:
+      - op: add
+        path: /spec/template/spec/kubeadmConfigSpec/initConfiguration/nodeRegistration/taints
+        value: []
```

Comment:
this just needs to be an empty value? can you help explain why?

Reply:
Taints has to be empty as the control plane will have a master schedule taint, so we are setting it to empty such that the single node in cluster would be able to schedule the workloads.

Reply:
Ok, that makes sense